src/migrations/1617092420994-AddAssets.ts

```diff
@@ -6,7 +6,7 @@ export class AddAssets1617092420994 implements MigrationInterface {
     public async up(queryRunner: QueryRunner): Promise<void> {
         await queryRunner.query(`CREATE TABLE "did_document_entity" ("id" character varying NOT NULL, "service" jsonb NOT NULL, "authentication" jsonb NOT NULL, "created" character varying, "delegates" text array, "proof" jsonb, "publicKey" jsonb NOT NULL, "updated" character varying, "@context" character varying NOT NULL, "logs" character varying NOT NULL, CONSTRAINT "PK_d96048f4c93be203eeff05ef404" PRIMARY KEY ("id"))`);
         await queryRunner.query(`CREATE TABLE "asset" ("id" text NOT NULL, "owner" character varying NOT NULL, "offeredTo" character varying, "createdAt" character varying NOT NULL, "updatedAt" character varying NOT NULL, "documentId" character varying, CONSTRAINT "REL_c82497e54ba94a090f9c5f4bff" UNIQUE ("documentId"), CONSTRAINT "PK_1209d107fe21482beaea51b745e" PRIMARY KEY ("id"))`);
-        await queryRunner.query(`CREATE TYPE "assets_history_type_enum" AS ENUM('ASSET_CREATED', 'ASSET_OFFERED', 'ASSET_OFFER_CANCELED', 'ASSET_TRANSFERRED', 'ASSET_OFFER_REJECTED')`);
+        await queryRunner.query(`CREATE TYPE "assets_history_type_enum" AS ENUM('ASSET_CREATED', 'ASSET_OFFERED', 'ASSET_OFFER_CANCELED', 'OFFERED_FROM', 'ASSET_OFFER_REJECTED')`);
```

Comment:
I think would make sense to generate a new migration for change like this instead of editing an existing migration